assets/queries/terraform/gcp/vm_serial_ports_are_enabled_for_vm_instances/metadata.json

```
@@ -3,7 +3,7 @@
   "queryName": "Serial Ports Are Enabled For VM Instances",
   "severity": "MEDIUM",
   "category": "Insecure Configurations",
-  "descriptionText": "Check if VM instance enables serial ports",
+  "descriptionText": "Check if serial ports are enabled in Google Compute Engine VM instances",
```

For all "Serial Ports Are Enabled For VM Instances" queries:

```suggestion
  "descriptionText": "Google Compute Engine VM instances should not enable serial ports",
```

---

ok to all